src/client/array/dc_array.c

```diff
@@ -233,8 +233,13 @@ create_handle_cb(tse_task_t *task, void *data)
 		daos_obj_close_t *close_args;
 		tse_task_t *close_task;
 
-		daos_task_create(DAOS_OPC_OBJ_CLOSE, tse_task2sched(task),
-				 0, NULL, &close_task);
+		rc = daos_task_create(DAOS_OPC_OBJ_CLOSE, tse_task2sched(task),
+				      0, NULL, &close_task);
+		if (rc != 0) {
```

Comment:
(style)  suspect code indent for conditional statements (16, 25)

Reply:
Do you know why this is?  The indentation looks correct here but maybe you've got a tabs/spaces issue?

Reply:
Yes, there is an extra space in the two lines following this if statement.